app/views/incomeSources/cease/CeaseIncomeSources.scala.html

```diff
@@ -70,39 +71,34 @@
         @h2(msg = getMessage("self-employment.h1"),
             optId = Some("self-employment-h1"))
 
-        @govukTable(Table(
-            head = Some(List(
-                HeadCell(
-                    content = Text(getMessage("table-head.business-name")),
-                    attributes = Map("id" -> "table-head-business-name")),
-                HeadCell(
-                    content = Text(getMessage("table-head.date-started")),
-                    attributes = Map("id" -> "table-head-date-started")),
-                HeadCell(
-                    content = Empty,
-                    attributes = Map("id" -> "empty1"))
-            )),
-            rows = sources.soleTraderBusinesses.zipWithIndex.map { case (business, index) => Seq(
-                TableRow(
-                    content = Text(business.tradingName),
-                    attributes = Map("id" -> s"table-row-trading-name-$index")),
-
-                TableRow(
-                    content = Text(business.tradingStartDate.toLongDate),
-                    attributes = Map("id" -> s"table-row-trading-start-date-$index")),
-
-                TableRow(
-                    content = HtmlContent(
-                        s"""<dd class="govuk-summary-list__actions">
-                                <a class="govuk-link" href='$ceaseSoleTraderBusinessURL'>
-                                    ${getMessage("cease")}<span class="govuk-visually-hidden">cease</span>
-                                </a>
-                            </dd>"""
-                    )
-                )
-            )},
-            attributes = Map("id" -> "sole-trader-businesses-table")
-        ))
+        <table class="govuk-table govuk-!-margin-bottom-3" id="sole-trader-businesses-table">
+            <thead class="govuk-table__head">
+                <tr class="govuk-table__row">
+                    <th class="govuk-table__header" scope="col" id="table-head-business-name">@getMessage("table-head.business-name")</th>
+                    <th class="govuk-table__header" scope="col" id="table-head-date-started">@getMessage("table-head.date-started")</th>
+                    <td class="govuk-table__header" id="empty0"></td>
```

Comment:
this might fail one of accessibility rule: => _id="empty0"_

Reply:
I thought I solved this accessibility issue by using the td tag rather than the th tag like for the other headings

Reply:
for heading we should use th in html table; but my concern was around using empty0 in the id attribute; lets leave this to a11y job;